packages/mobile/src/utils/generators.ts

```diff
@@ -1,19 +0,0 @@
-// adapted from https://www.promisejs.org/generators/
-export function promisifyGenerator<T>(generator: Generator<any, T, unknown>) {
```

Comment:
anywhere I can read about why this doesn't work with redux sagas?
mostly just curious

Reply:
There's no single place that explains this unfortunately, just small clues about how Saga actually works sprinkled throughout the docs and the github issues
https://redux-saga.js.org/